ckanext/data_qld_harvester/templates/package/snippets/resources_list.html

```diff
@@ -0,0 +1,27 @@
+{% if pkg.type == 'geoscience' %}
+<!-- No resources for geoscience datasets -->
+{% else %}
+    <section id="dataset-resources" class="resources">
```

Comment:
@ThrawnCA  could this be a super?

Reply:
Looks like yes. It's a direct copy-paste from the core template.